diff --git a/node_modules/collect-v8-coverage/index.js b/node_modules/collect-v8-coverage/index.js
index 08b7853..8dab3d4 100644
--- a/node_modules/collect-v8-coverage/index.js
+++ b/node_modules/collect-v8-coverage/index.js
@@ -26,9 +26,9 @@ class CoverageInstrumenter {
       'Profiler.takePreciseCoverage',
     );
 
-    await this.postSession('Profiler.stopPreciseCoverage');
+    // await this.postSession('Profiler.stopPreciseCoverage');
 
-    await this.postSession('Profiler.disable');
+    // await this.postSession('Profiler.disable');
 
     // When using networked filesystems on Windows, v8 sometimes returns URLs
     // of the form file:////<host>/path. These URLs are not well understood
